newIDE/app/src/ResourcesList/LocalPiskelBridge.js

```diff
@@ -47,20 +56,34 @@ export const openPiskel = ({
   ipcRenderer.removeAllListeners('piskel-changes-saved');
   ipcRenderer.on(
     'piskel-changes-saved',
-    (event, outputResources, newAnimationName) => {
+    (event, outputResources, newAnimationName, metadata) => {
+      const newMetadata =
+        'data' in metadata ?
```

Comment:
You can simply write `metadata.data ?` It's statically analyzable (i.e: understood by flow) and more concise.
This is because a non existing property will return `undefined` which is falsy (https://developer.mozilla.org/en-US/docs/Glossary/Falsy).